matter/wifi/wf200/host_if.cpp

```diff
@@ -393,7 +393,7 @@ static void sl_wfx_disconnect_callback(uint8_t *mac, uint16_t reason)
  *****************************************************************************/
 static void sl_wfx_start_ap_callback(uint32_t status)
 {
-  if (status == 0) {
+  if (status == STATUS_0) {
```

Comment:
what is a status of 0 what does it mean. Good bad? :)

Reply:
done